src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
+
+public BasePage(WebDriver givenDriver){
+    driver = givenDriver;
+    wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    actions = new Actions(driver);
+    PageFactory.initElements(driver, this);
+}
+public void doubleClick(WebElement locator){
+wait.until(ExpectedConditions.elementToBeClickable(locator));
+actions.doubleClick().perform();
+}
+}
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)